src/components/molecules/card/card.scss

```
@@ -0,0 +1,48 @@
+@use "../../variables" as *;
+
+.card {
+  position: relative;
+  .card-img {
+    overflow: hidden;
+    .triangle {
+      position: absolute;
+      border-bottom: 150px solid transparent;
+      border-left: 150px solid $white;
+      z-index: 1;
+    }
+    .card-img-top {
+      transition: all 0.5s 0s ease;
+      &:hover {
+        transform: scale(1.2);
+      }
+      @media screen and(max-width: $breakpoint-md) {
+        pointer-events: none;
+      }
```

```suggestion
      @media screen and(max-width: $breakpoint-md) {
        &:hover {
          transform: scale(1.2);
        }
      }
```
こんな感じでも書けると思います。もちろんこのままでも問題ないです。

---

ありがとうございます。次回から参考させていただきます。

---

今見返したらこれじゃダメかもしれないです。失礼しました 🙏

---

了解しました。わざわざありがとうございます。